cloudinit/config/schema.py

```diff
@@ -909,6 +910,13 @@ def _get_examples(meta: MetaSchema) -> str:
     return rst_content
 
 
+def _get_activate_by_schema_keys_doc(meta: MetaSchema) -> str:
+    if not meta.get("activate_by_schema_keys"):
+        return ""
+    schema_keys = ", ".join(meta["activate_by_schema_keys"])
+    return f"**Activate only if given keys:** {schema_keys}\n\n"
```

Comment:
Let's mark up the key names with ``<key_name>`` to align with other doc config key references. It makes readthedocs decorate it in red.

```suggestion
    schema_keys = ", ".join(meta["activate_by_schema_keys"])
    schema_keys = ", ".join(
        f"``{k}``" for k in meta["activate_by_schema_keys"]
    )
    return f"**Activate only on keys:** {schema_keys}\n\n"
```